docs/source/alfacase_definitions/PumpEquipmentDescription.txt

```diff
@@ -22,7 +23,7 @@
             user_defined_esp_table: \ :class:`TablePumpDescription <TablePumpDescription>`\  = TablePumpDescription()
             esp_parameters: \ :class:`EspParameters <alfasim_sdk._internal.constants.EspParameters>`\  = EspParameters.UserDefined
             esp_viscosity_model: \ :class:`PumpViscosityModel <alfasim_sdk._internal.constants.PumpViscosityModel>`\  = PumpViscosityModel.NoModel
-            esp_thermal_efficiency_model: \ :class:`PumpThermalEfficiencyModel <alfasim_sdk._internal.constants.PumpThermalEfficiencyModel>`\  = PumpThermalEfficiencyModel.Constant
```

Comment:
Why it was removed? It is currently being used to enable the thermal efficiency model at master.

Reply:
@rafacerq yet i see this. So this attribute is used for all pump types, so they now will be renamed by ```thermal_efficiency_model```

Reply:
Ok!